src/elements/content-sidebar/__tests__/BoxAISidebar.test.tsx

```diff
@@ -423,4 +424,14 @@ describe('elements/content-sidebar/BoxAISidebar', () => {
             expect(mockSendQuestion).not.toHaveBeenCalled();
         },
     );
+
+    test('should call onUserInteraction when user takes action', async () => {
```

Comment:
Should we cover when `userHasInteracted`? Or is this covered in `box-ai-content-answers`?

Reply:
This test already covers that. `onUserIntentToUseAI` is called with `userHasInteracted = true` when user types in prompt. The logic of setting this param is inside `content-answers`.